lualib/resty/events/worker.lua

```diff
@@ -143,118 +256,22 @@ function _M:communicate(premature)
         return
     end
 
-    self._connected = true
     log(DEBUG, _worker_id, " on (", listening, ") is ready")
```

Comment:
We can use the `log_if_not_exiting` here, and then remove https://github.com/Kong/lua-resty-events/pull/45/files#diff-d0bbfa33cfe00d7863c292aaefa4fb096eeddaa8f0a24539bb90b7ffd56c60c5R246?